webapp/packages/plugin-devtools/src/DevToolsService.ts

```diff
@@ -15,6 +15,7 @@ import { LocalStorageSaveService } from '@cloudbeaver/core-settings';
 interface IDevToolsSettings {
   enabled: boolean;
   distributed: boolean;
+  codemirror: boolean;
```

Comment:
move this setting to `plugin-sql-editor-new` (`enabled`)
and switch it here in the `DevTools`